pkgs/applications/video/lossless-cut-bin/default.nix

```
@@ -0,0 +1,31 @@
+{ pkgs, stdenvNoCC, lib }:
+let
+  version = "3.33.1";
+  callBin = pathExpression:
+    (pkgs.callPackage pathExpression { inherit version; }).overrideAttrs
+    (oldAttrs: { meta = meta // (oldAttrs.meta or { }); });
+  passthru = {
+    appimage = callBin ./appimage.nix;
+    dmg = callBin ./dmg.nix;
+    windows = callBin ./windows.nix;
+  };
+  meta = with lib; {
+    description = "The swiss army knife of lossless video/audio editing";
+    homepage = "https://mifi.no/losslesscut/";
+    license = licenses.mit;
+    maintainers = with maintainers; [ ShamrockLee ];
```

```suggestion
    description = "The swiss army knife of lossless video/audio editing";
    homepage = "https://mifi.no/losslesscut/";
    license = licenses.mit;
    maintainers = with maintainers; [ ShamrockLee ];
```
Why not put these into the override below, get rid of `callBin`